public/components/overview/mitre_attack_intelligence/module_mitre_attack_intelligence_resource.tsx

```diff
@@ -11,13 +11,45 @@
  * Find more information about this on the LICENSE file.
  */
 
-import React, { useCallback, useState } from 'react';
+import React, { useCallback, useState, useEffect } from 'react';
 import { TableWzAPI } from '../../../components/common/tables';
+import { WzRequest } from '../../../react-services';
+import { ModuleMitreAttackIntelligenceFlyout } from './resource_detail_flyout';
+
+
+export const ModuleMitreAttackIntelligenceResource = ({ label, searchBarSuggestions, apiEndpoint, tableColumns, initialSortingField, resourceFilters }) => {
 
-export const ModuleMitreAttackIntelligenceResource = ({ label, searchBarSuggestions, apiEndpoint, tableColumns, detailsProperties, initialSortingField = 'name', resourceFilters }) => {
   const [isDetailsOpen, setIsDetailsOpen] = useState(false);
   const [details, setDetails] = useState(null);
 
+  useEffect(() => {
+    const urlParams = new URLSearchParams(location.href);
+    const redirectTab = urlParams.get('tabRedirect');
+    const idToRedirect = urlParams.get("idToRedirect")
+    if(redirectTab && idToRedirect){
+      const endpoint = `/mitre/${redirectTab}?q=references.external_id=${idToRedirect}`;
+      getMitreItemToRedirect(endpoint)
+      urlParams.delete('tabRedirect')
+      urlParams.delete('idToRedirect')
+      window.history.pushState({},document.title,'#/overview/?tab=mitre')
+    }
+  },[]);
+
+
+  const getMitreItemToRedirect = (endpoint) => {
+       WzRequest.apiReq('GET', endpoint, {}).then(res => {
```

Comment:
Maybe you could replace the `.then/.catch` block with `try/catch` that is trending to use.